### PR TITLE
first bash at episode-ing the code

### PR DESCRIPTION
**Pull request by CaitlinBate**
same code that was already tested by NT and BD, and put into the episode-verbage that was already approved in the OTW tickets. so hoping this is going to be an easy merge

### CODE REVIEW

File: _episodes/01-metadata.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+---
+## Reporting Data to a OTN Node
+
+As researchers who are part of the MigraMar Network, you are encouraged to register your projects and report your data and metadata in a timely manner to your Data Manager. This will benefit all researchers in the region through the database's detection-matching system.
+
+This presentation **LINK TO PPT HERE** will cover some of the following topics.
+
+You are encouraged to read the [OTN FAQs Page](https://members.oceantrack.org/faq) for more information.
+
+### How to register with the MigraMar Database?
```
Comment (jackVanish) [inline _episodes/01-metadata.md:18]: Should be no question mark here.

File: _episodes/01-metadata.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+### What is the benefit of registering with the MigraMar Database?
+
+MigraMar and affiliated networks provide automated cross-referencing of your detection data with other tags in the system to help resolve "mystery detections" and provide detection data to taggers in other regions. MigraMar's Data Manager will also extensively quality control your submitted metadata for errors to ensure the most accurate records possible are stored in the database. MigraMar's database and Data Portal website are excellent places to archive your datasets for future use and sharing with collaborators. We offer pathways to publish your datasets with [OBIS](https://obis.org/), and via open data portals like ERDDAP, GeoServer etc. The data-product format returned by MigraMar is directly ingestible by analysis packages such as [glatos](https://github.com/ocean-tracking-network/glatos) and [resonATe](https://gitlab.oceantrack.org/otndc/resonate) for ease of analysis. OTN offers support for the use of these packages and tools.
```
Comment (jackVanish) [inline _episodes/01-metadata.md:24]: Should 'mystery detections' be in quotes?

File: _episodes/01-metadata.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+questions:
+    - "What is the Ocean Tracking Network?"
+    - "How does your local telemetry network interact with OTN?"
+    - "What methods of data analysis will be covered?"
+---
+## Reporting Data to a OTN Node
+
+As researchers who are part of the MigraMar Network, you are encouraged to register your projects and report your data and metadata in a timely manner to your Data Manager. This will benefit all researchers in the region through the database's detection-matching system.
+
+This presentation **LINK TO PPT HERE** will cover some of the following topics.
```
[the remark this answers is not in the document]
Comment (CaitlinBate) [reply _episodes/01-metadata.md:14]: added later, dont have a ppt ready yet haha

**Review by jackVanish — CHANGES_REQUESTED** [review]
This looks solid but for the typos I noticed in episode 01. Other than that, I agree that the code works and I didn't notice any obvious typos throughout, so I think if it displays right we're probably fine.

File: _episodes/01-metadata.md (hunk 2 above)
Comment (CaitlinBate) [reply _episodes/01-metadata.md:24]: its OTN-wording for unknown tags. other groups call them "orphan tags" "unknown fish" etc. so i think it should stay in quotes

**Comment by CaitlinBate** [comment]
fixes here https://github.com/ocean-tracking-network/migramar-student-workshop-2022/pull/7/commits/487a2dc2f0dc7e598f91a1f907e24565db0fc984

**Comment by CaitlinBate** [comment]
merging so i can check the formatting on the website

**Pull request merged**